package.json

```diff
@@ -44,17 +44,14 @@
     "gulp-sequence": "0.4.6",
     "gulp-shell": "0.6.3",
     "mocha": "3.2.0",
-    "yeoman-assert": "3.0.0",
-    "yeoman-test": "1.6.0"
+    "yeoman-assert": "3.1.0",
+    "yeoman-test": "1.7.0"
   },
   "scripts": {
     "pretest": "eslint .",
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
     "test": "mocha test/* --slow 0 --reporter spec"
   },
-  "license": "Apache-2.0",
-  "bugs": {
```

Comment:
why is this removed?

Reply:
we have duplicate bug entry in the json file. So, have removed the one of them